src/components/Nav/Nav.scss

```
@@ -1,3 +1,52 @@
+/* stylelint-disable selector-class-pattern */
 * {
   box-sizing: border-box;
+  margin-left: 10px;
+}
+.navContainer {
+  display: flex;
+  position: fixed;
+  z-index: 1;
+  justify-content: space-between;
+  width: 100%;
+  height: 100px;
+  border-bottom: 1px solid lightgray;
+  background-color: white;
+  line-height: 100px;
+  .mainLogo {
+    margin-left: 30px;
+    font-size: 30px;
+    font-weight: 700;
+  }
+  .navMenu {
+    display: flex;
+    justify-content: space-between;
```

```suggestion
.navContainer {
  display: flex;
  position: fixed;
  z-index: 1;
  justify-content: space-between;
  width: 100%;
  height: 100px;
  border-bottom: 1px solid lightgray;
  background-color: white;
  line-height: 100px;
  
  .mainLogo {
    margin-left: 30px;
    font-size: 30px;
    font-weight: 700;
  }
  
  .navMenu {
    display: flex;
    justify-content: space-between;
```
1. 가독성을 위해 줄바꿈을 추가해주세요.
2. navContainer에 line-height 값은 꼭 필요한 값일까요? 필요한 값이라면 그대로 사용하셔도 됩니다.